### PR TITLE
Update outdated dependencies

### PR DESCRIPTION
**Pull request by magopian**
Fixes #512, #555, #534

### CODE REVIEW

**Comment by leplatrem** [comment]
Don't we have a lock file?

**Comment by magopian** [comment]
We don't. Yet. cf #534. Also this is not done, I'm now trying to update all the dependencies that have major version changes.

**Comment by magopian** [comment]
This should now be good to merge, not much to review I guess...

File: package.json [hunk 1]
```diff
@@ -36,9 +36,9 @@
     "atob": "^2.0.3",
     "babel-polyfill": "^6.23.0",
     "babel-runtime": "^6.11.6",
-    "bootstrap": "^3.3.6",
+    "bootstrap": "^4.1.3",
```
Comment (glasserc) [inline package.json]: This seems a bit intense to me... don't we have to change markup for this?

Comment (magopian) [reply package.json]: Yeah, you're right, I'll revert that, I'm not going to dig into visual changes, at least for now ;)

**Pull request merged**